connection_pool/round_robin.go

```diff
@@ -46,8 +46,10 @@ func (r *RoundRobinStrategy) DeleteConnByAddr(addr string) *tarantool.Connection
 	r.conns = append(r.conns[:index], r.conns[index+1:]...)
 	r.size -= 1
 
-	for index, conn := range r.conns {
-		r.indexByAddr[conn.Addr()] = index
+	for k, v := range r.indexByAddr {
```

Comment:
I don't got it. A in-built function for what?

Reply:
To remove an item from the middle of the slice

Reply:
I think that it is a in-built in way:
```Go
r.conns = append(r.conns[:index], r.conns[index+1:]...)
```
Sorry, but I can't got what the alternative is.

Reply:
`slices.Delete`

https://pkg.go.dev/golang.org/x/exp/slices#Delete

Reply:
https://pkg.go.dev/golang.org/x/exp#section-readme

```Warning: Packages here are experimental and unreliable. Some may one day be promoted to the main repository or other subrepository, or they may be modified arbitrarily or even disappear altogether.```

Let's do without it, since it's not critical.

Reply:
Yeah, it's too early